packages/flutter/test/material/input_decorator_test.dart

```diff
@@ -3801,6 +3882,93 @@ void main() {
     },
   );
 
+  testWidgets(
+    'InputDecorator OutlineBorder focused center floating label with icon',
+    (WidgetTester tester) async {
+      Widget buildFrame(TextDirection textDirection) {
+        return MaterialApp(
+          home: Scaffold(
+            body: Container(
+              padding: const EdgeInsets.all(16.0),
+              alignment: Alignment.center,
+              child: Directionality(
+                textDirection: textDirection,
+                child: const RepaintBoundary(
+                  child: InputDecorator(
+                    isFocused: true,
+                    isEmpty: true,
+                    decoration: InputDecoration(
+                      icon: Icon(Icons.insert_link),
+                      floatingLabelAlignment: FloatingLabelAlignment.center,
+                      labelText: 'primaryLink',
+                      hintText: 'Primary link to story',
+                      border: OutlineInputBorder(),
+                    ),
+                  ),
+                ),
+              ),
+            ),
+          ),
+        );
+      }
+
+      await tester.pumpWidget(buildFrame(TextDirection.ltr));
+      await expectLater(
+        find.byType(InputDecorator),
+        matchesGoldenFile('input_decorator.outline_icon_center_floating_label.ltr.png'),
```

Comment:
Any updates on this?

Reply:
Sorry I missed your response here.  Let's remove the goldens.

Usually I try to only use goldens when there is no other way to test something, because of their maintenance burden.  I've seen so many goldens fail unnecessarily when we make a tiny change to antialiasing or something like that.